improver_tests/acceptance/SHA256SUMS

```diff
@@ -100,13 +100,12 @@ c72e48ae88fb3a9c909323376c20d9bcaecec5e1b55a42accdab7201ce8d7cc8  ./estimate-emo
 a7bd5d01ed80297fb5b28be9a308fd8de20c162a87220be5e59c8ac3ffa800f0  ./estimate-emos-coefficients/normal/history/20170604T0300Z-PT0012H-temperature_at_screen_level.nc
 2da5282374a1c36e14de78891bfc49b1dd4a91b57fb4c4f7ad2bc59564acf313  ./estimate-emos-coefficients/normal/kgo.nc
 cb9179b0aca8d1f8cccf858f71c51fccddda967d93cf041b3c2c6d4d7db68c03  ./estimate-emos-coefficients/normal/land_only_kgo.nc
-bbb80afc1a2770b3a1a3cb666fe01e7ef7c2e9fc993db0c8688f2563927f8eda  ./estimate-emos-coefficients/normal/realizations/with_statsmodels_kgo.nc
-0cd25c95b4d3710c9ca21409d9143cc9249aa9ef9e3e9d0bd166fa2cd89bedd6  ./estimate-emos-coefficients/normal/realizations/without_statsmodels_kgo.nc
+bbb80afc1a2770b3a1a3cb666fe01e7ef7c2e9fc993db0c8688f2563927f8eda  ./estimate-emos-coefficients/normal/realizations/kgo.nc
 42fe8daffb997848086cb08b3d39dd366b561bb1c24fc7ed3e2e54737d772b9a  ./estimate-emos-coefficients/normal/sites/history/20201209T1500Z-PT0012H00M-temperature_at_screen_level.nc
 3cda6a3244db09530585454d323ec13bde6b4cfae24940a8b069469f47aec60c  ./estimate-emos-coefficients/normal/sites/history/20201210T1500Z-PT0012H00M-temperature_at_screen_level.nc
 e5dab98673996a3663093a0353b4efac7910b8176921bd0fbf10aa0f53ae5800  ./estimate-emos-coefficients/normal/sites/history/20201211T1500Z-PT0012H00M-temperature_at_screen_level.nc
 4ff9304b3eca75282ba8aa4a18e0de3404d8510438adccfec7fb20255bfd46c0  ./estimate-emos-coefficients/normal/sites/point_by_point/kgo.nc
-ce9102f00f23256d401eb4cedaa3c70e6a9fcab4dea373ed0f83989215628fa9  ./estimate-emos-coefficients/normal/sites/point_by_point/realizations_kgo.nc
+e9b48f6805af8db049b1b786ee539589ff6186c619a45e9c14cc22a008276f2a  ./estimate-emos-coefficients/normal/sites/point_by_point/realizations_kgo.nc
```

Comment:
Why has this test changed?

Reply:
This is because the `test_normal_realizations_point_by_point_sites` acceptance test used to have a `acc.skip_if_statsmodels` decorator, so the output from this test was only generated if statsmodels wasn't present. As statsmodels is always present now, the output from this test has changed due to the use of statsmodels to estimate an initial guess. Previously, as this test was only run without statsmodels, a default initial guess would have been used, which would lead to different values for the coefficients being estimated.

Reply:
I think this is ok, thanks for investigating